### ISSUE TITLE
Update for 2020?

### ISSUE THREAD

**Issue by swamidass** [greenelab/deep-review#1021]
Are we doing an updated review for 2020?

**Comment by agitter** [comment]
@swamidass yes, there are some updates underway.  We've backed off calling it a 2020 update because there is not a fixed timeline.  Instead we're referring to the updated version generically as "version 2.0" and updated the manuscript to indicate new v2 authors.  My current assessment is that we welcome more updates from new authors or v1 authors and will (slowly) work to review them, but none of us are pushing the v2 update as a top priority.

#959 describes some of the updates that have been made for v2.  There are additional open pull requests that are being reviewed.  There is also discussion of a major reorganization in #1005.

@cgreene created issues to discuss updates to specific sections: #962 #963 #964 #965 #966, #967, #968

**Comment by swamidass** [comment]
Thanks for the update. What would be the best way for me to help with version 2.0?

**Comment by cgreene** [comment]
To me, the first thing we would do is simply move text around. This suggestion from @manolik seemed like a good solution. That would give us the organizational structure to then start going through section by section (and inviting others with particularly expertise to do so) and updating to the current state of the field.

https://github.com/greenelab/deep-review/issues/1005
